scripts/qa_crowbarsetup.sh

```diff
@@ -1450,7 +1450,7 @@ EOF
 
     # avoid kernel update
     zypper al kernel-default
-    zypper -n dup --no-recommends -r Cloud -r cloudtup || zypper -n dup --no-recommends -r Cloud
+    zypper -n dup --no-recommends --no-allow-downgrade -r Cloud -r cloudtup || zypper -n dup --no-recommends --no-allow-downgrade -r Cloud
```

Comment:
we could reduce duplication by introducing`zypperdup="zypper -n dup --no-recommends --no-allow-downgrade"`